mobile_config/src/sub_dao_epoch_reward_info.rs

```diff
@@ -0,0 +1,113 @@
+use crate::EpochPeriod;
+use chrono::{DateTime, Utc};
+use file_store::traits::{TimestampDecode, TimestampEncode};
+use helium_proto::services::sub_dao::SubDaoEpochRewardInfo as SubDaoEpochRewardInfoProto;
+use rust_decimal::prelude::*;
+use std::ops::Range;
+
+#[derive(Clone, Debug)]
+pub struct ResolvedSubDaoEpochRewardInfo {
+    pub epoch: u64,
+    pub epoch_address: String,
+    pub sub_dao_address: String,
+    pub epoch_period: Range<DateTime<Utc>>,
+    pub epoch_emissions: Decimal,
+    pub rewards_issued_at: DateTime<Utc>,
+}
+
+#[derive(Clone, Debug)]
+pub struct RawSubDaoEpochRewardInfo {
+    epoch: u64,
+    epoch_address: String,
+    sub_dao_address: String,
+    rewards_issued: u64,
+    delegation_rewards_issued: u64,
+    rewards_issued_at: DateTime<Utc>,
+}
+
+impl TryFrom<RawSubDaoEpochRewardInfo> for SubDaoEpochRewardInfoProto {
```

Comment:
This cannot fail, can we convert it to a regular `From`?

Reply:
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393